forest/daemon/src/daemon.rs

```diff
@@ -383,16 +364,10 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
         _ = ctrlc_oneshot => {}
     }
 
-    let keystore_write = tokio::task::spawn(async move {
```

Comment:
Is this removed intentionally?

Reply:
Yes, the keystore is flushed on writes so there's no need to flush it here.